app/assets/javascripts/libs/utils.js

```diff
@@ -528,3 +528,16 @@ const Utils = {
 };
 
 export default Utils;
+
+export function enforce<A, B>(fn: A => B): (?A) => B {
```

Comment:
Yeah, it's a bit inconsistent unfortunately (my bad!). Ideally, I'd like to convert everything to a normal `export function ...() {}` notation, as the explicit Utils wrapper doesn't really yield a benefit in my opinion (it's only an additional indention layer). I moved the functions to the top so that they are near the other functions. I'll create an issue for changing the structure of the entire file, if you agree. Shouldn't be a big change.